### DISCUSSION TITLE
Want to compile BLtouch 5x5

### DISCUSSION THREAD

**Discussion by Gil80** [Jyers/Marlin#1192] [Q&A]
Hi.
I can't figure out what do I need to change to compile bltouch 5x5. I know there is a .bin file ready to download, but there are some changes I need to make and in addition I want the 5x5.

thanks.

I think it's this: `#define GRID_MAX_POINTS_X 5` correct me if I'm wrong please.

**Comment by Cal-LF** [comment]
Yes, I change mine to 4 for 16 check points.

**Reply by Gil80** [reply]
thanks!
